unified-codes/apps/web/src/reducers/ExplorerReducer.ts

```diff
@@ -0,0 +1,47 @@
+import { EXPLORER_ACTIONS, IExplorerAction, IExplorerUpdateVariablesAction, IExplorerFetchFailureAction, IExplorerFetchSuccessAction } from '../actions';
+import { IExplorerState } from '../types';
+
+const initialState = (): IExplorerState => {
+  return {
+    loading: false,
+    variables: {},
```

Comment:
`variables` doesn't work for me  - such a vague name. I thought `entities` was fine though, and that's also vague. So is `data` and that's also fine 🤷‍♂️